logger.js

```diff
@@ -176,19 +176,16 @@ logger.log_respond = function (retval, msg, data) {
 };
 
 logger.set_loglevel = function (level) {
-
-    if (!level) return;
```

Comment:
instead of removing this, I'd replace it with an `if (level === undefined || level === null)`

Reply:
Your current Travis test failure answers: why?